src/app/libmain.cxx

```diff
@@ -70,13 +70,13 @@ static RpmOstreeCommand commands[] = {
   { "reload", static_cast<RpmOstreeBuiltinFlags>(0),
     "Reload configuration",
     rpmostree_builtin_reload },
+  /* Defined in Rust */
   { "usroverlay", static_cast<RpmOstreeBuiltinFlags>(RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT),
-    "Apply a transient overlayfs to /usr",
-    rpmostree_builtin_usroverlay },
+    "Apply a transient overlayfs to /usr", NULL },
   /* Let's be "cognitively" compatible with `ostree admin unlock` */
+  /* Defined in Rust */
   { "unlock", static_cast<RpmOstreeBuiltinFlags>(RPM_OSTREE_BUILTIN_FLAG_HIDDEN),
-    NULL,
-    rpmostree_builtin_usroverlay },
+    NULL, NULL },
```

Comment:
Since it doesn't show up in `--help` anyway, we can drop this entirely since the only reason to have it here is for the help output.